src/app/jscaip/Direction.ts

```diff
@@ -12,6 +12,9 @@ export class Vector implements ComparableObject {
     }
     public constructor(public readonly x: number,
                        public readonly y: number) {}
+    public toString(): string {
+        return this.x + ',' + this.y;
```

Comment:
un petit espace après la virgule et ... why not their name rather ?

Reply:
le nom de... ? c'est quoi le nom de (5, -3) ? :D

Reply:
Ok comme c'est Vector juste fait (x, y) ?